docs/en/observability/synthetics-recorder.asciidoc

```diff
@@ -0,0 +1,136 @@
+
+[discrete]
+[[synthetics-recorder-edit-or-delete-recorded-actions]]
+=== Edit or delete recorded actions
+
+You can fine-tune a journey by editing actions that were generated by the recorder.
+You can't change the type of command (for example, "click" or "navigate"), but you can change the value that is passed to the command.
+
+To edit an action:
+
+. Hover over an action and click the pencil icon that appears.
+. Edit the value as needed.
+. Click **Save**.
+
+To delete an action:
+
+. Hover over the action you want to delete and click the three dots for more options.
+. Click **Delete action**.
+
+IMPORTANT: If you changed or deleted any actions to ensure the journey still works, it's especially important to <<synthetics-recorder-test-the-journey, test the journey>>.
+
+[discrete]
+[[synthetics-recorder-add-assertions]]
+=== Add assertions
+
+Assertions can play an important role in effective synthetic journeys by making determinations about the state of the page you are testing.
+This can include checking if an element is visible or checking the contents of a text field.
+You can't generate an assertion just from interacting with the browser window.
+Instead, you can add assertions between generated actions.
+
+To add an assertion:
+
+. Find the generated action that should be done right before you want to assert a condition.
+. Hover over that action and click the three dots for more options.
+. Click **Add assertion**. This will add a new "assert" action in the UI.
+. Provide the type of assertion, selector, and value.
+. Click **Save**.
+
+IMPORTANT: If you added any assertions after you've finished recording to ensure the journey still works, it's especially important to <<synthetics-recorder-test-the-journey, test the journey>>.
+
+[discrete]
+[[synthetics-recorder-test-the-journey]]
+== Test the journey
+
+At any point during or after the recording process concludes, you can test your script.
+
+When you click the **Test** button, Elastic Synthetics will run the journey.
+As the test runs, the recorder will display results on a per-step basis.
+If there are any errors that prevent the journey from running, the recorder will display the relevant error message to help you debug.
+
+IMPORTANT: If you paused recording, updated actions, or added assertions manually in the recorder it is especially important that you test the journey to  verify that the actions work in sequence.
+
+[discrete]
+[[synthetics-recorder-export]]
+== Export
+
+When you are satisfied with journey you've created, you can export it from the recorder.
+
+Click **Export** to view the final journey code.
+From there you can use the code by:
+
+* Copy and pasting code containing all steps into a new or existing <<synthetic-monitor-choose-project, synthetics project>> or an <<synthetics-inline-journey, inline monitor>>.
+* Click **Export** to save a JavaScript file containing all steps.
```

Comment:
We don't really say much more about why this would be used, or exists as an option (file can be dragged onto add monitor).  Probably ok as we're updating this primary flow, but might be worth adding a note something for it?